Algorithms/Sort/InsertionSort/README.md

```diff
@@ -0,0 +1,40 @@
+# Insertion Sort
+Insertion sort is a simple sorting algorithm that builds the final sorted array one element at a time. It is much less efficient on large lists than more advanced algorithms such as `QuickSort`, `MergeSort`, or `HeapSort`. However, it performs well for small datasets or lists that are nearly sorted.
+
+## Pseudocode
+```plaintext
+function insertionSort(array):
+	n = array.length()
+    for i = 1 to n - 1:
+		key = array[i]
+    	j = i - 1
+
+    	# Move elements greater than key to one position ahead
+    	while j >= 0 and array[j] > key:
+        	array[j + 1] = array[j]
+        	j = j - 1
+
+    	# Place the key at its correct position
+    	array[j + 1] = key
+```
+
+## Time & Space Complexity
+|Case        | Time          | Space |
+|------------|----------------:|-----|
+|Best        | Ω(n)| Ω(1)|
+|Average     | Θ(n<sup>2</sup>)| Θ(1)|
+|Worst       | O(n<sup>2</sup>)| O(1)|
```

Comment:
Merging row feature is not available in GitHub Markdown.

Reply:
I will do it using raw html

Reply:
don't use raw HTML

Reply:
So... What you want to use then? It is the only way to do that!

Reply:
Lets leave that, and do the commit without merging, just fix the padding spaces

Reply:
The issue is that you're using `tab` instead of `4 spaces` here and there. That is causing these inconsistencies. Make sure those are spaces and not tab characters in there.